### ISSUE TITLE
Task-based fMRI: Tool to explore group differences

### ISSUE THREAD

**Issue by gifuni** [mtl-brainhack-school-2019/Anthony-Gifuni-repo#1]
Hi @pbellec @emdupre @illdopejake @alexapichet ,

My projects include a resting-state sequence and 2 task-based fMRI sequences collected with adolescents divided into 3 groups (N~30 in each group):
* Adolescents with depression who attempted suicide
* Adolescents with depression who did *not* attempt suicide
* Healthy controls

Several tools exist for analysis exists of such task data, but it would be interesting to combine the task and resting-state data in an all-encompassing analysis, or at least start working with tools that allow such analysis. 

My initial idea is to examine functional connectivity during tasks, examine how tasks alter FC and examine how group status alters these relationships. What are your first thoughts or go-to things for that kind of analysis?? Anything would be immensely welcome :)

Thanks!

**Comment by alexapichet** [comment]
Hello! 
Nice project :) 
With 3 functional scans for each participant, maybe it could be nice to try to fingerprint your participants (Finn et al, 2015) ... ?
Could be a realistic exercise to code within the next weeks I think with all the help around!

**Comment by emdupre** [comment]
My first thought for creating FC matrices to compare (across tasks or groups, or to fingerprint participants ?) would be [Nilearn](https://nilearn.github.io).

The [tutorial @illdopejake gave last week](https://github.com/neurodatascience/NeuroDataSci-course-2019/blob/master/content/day4/PM/ML_Regression_Tutorial.ipynb) could be a really nice jumping off point  !

Let me know if I can help ! :sparkles: :rocket:

**Comment by illdopejake** [comment]
I agree that nilearn is a good place to start. We can discuss in detail during the breakout session. The nilearn tutorials @emdupre linked to are a fantastic place to start in terms of getting used to interacting with both task and resting state data with python.

But I think you would benefit from a very well-defined study design. You have a really cool sample and a lot of data, but not so many subjects. So it's important to think carefully about how you will analyze and validate the data. Are there two runs of the same tasks, or two different tasks? What is your reason for wanting to look at connectivity during task rather than activation? (There is no right answer I'm just asking). 

Lots to think about, but good to start with something fairly straightforward and then get more complicated from there.
